app/Http/Controllers/LegacyController.php

```diff
@@ -49,11 +49,10 @@ public function index(Request $request, Session $session)
         $vars['page'] = basename($vars['page'] ?? '');
         if ($vars['page'] && is_file("includes/html/pages/" . $vars['page'] . ".inc.php")) {
             require "includes/html/pages/" . $vars['page'] . ".inc.php";
-        } elseif (Config::has('front_page') && is_file('includes/html/' . Config::get('front_page'))) {
```

Comment:
I think we should continue to support custom front pages if I'm honest. Whilst the other ones we provide might not work, people might have created their own. However we should have them place the custom files in a location outside of git.